tests/13-basic-attrs.c

```diff
@@ -108,6 +108,21 @@ int main(int argc, char *argv[])
 		goto out;
 	}
 
+
+	expected = 1;
+	rc = seccomp_attr_set(ctx, SCMP_FLTATR_SPEC_ALLOW, expected);
+	if (rc == -EOPNOTSUPP)
+		expected = 0;
```

Comment:
Since you are forcing the API level to "4" above, this should always be supported regardless of the underlying host kernel.  You should be able to just copy/paste the checks used for the other attributes (the "expected" stuff shouldn't be needed).

Reply:
@tych0 I'm guessing you simply missed this comment?

Also, you should update the Python variant of this test too.